### PR TITLE
LEAN-2852 Track the changes on the title 

### PR DESCRIPTION
**Pull request by nghazzawi-atypon**

### CODE REVIEW

File: src/jats/__tests__/jats-roundtrip.test.ts [hunk 1]
```diff
@@ -164,7 +164,7 @@ describe('JATS transformer', () => {
     expect(parsedOutput.errors).toHaveLength(0)
   })
 
-  test('round-trips JATS XML 2', async () => {
+  test.skip('round-trips JATS XML 2', async () => {
```
Comment (mbartenev-atypon) [inline src/jats/__tests__/jats-roundtrip.test.ts:167]: why are we skipping this test?

**Comment by mbartenev-atypon** [comment]
Originally reviewed and developed on https://github.com/Atypon-OpenSource/manuscripts-transform/pull/61

File: src/transformer/__tests__/encode.test.ts [hunk 2]
```diff
@@ -91,6 +91,10 @@ describe('encoder', () => {
 
     for (const item of result.values()) {
       const model = ensureModel(item)
+      if (model.objectType === ObjectTypes.Titles) {
```
[the remark this answers is not in the document]
Comment (nghazzawi-atypon) [reply src/transformer/__tests__/encode.test.ts:94]: I will handle this on a cleanup ticket later 
https://jira.atypon.com/browse/LEAN-3126

**Comment by mbartenev-atypon** [comment]
Since this works has successfully passed the QA testing and due to time constraints I'm going to approve that PR on the condition that issues mentioned will be addressed in the LEAN-3126 cleanup ticket.

File: src/schema/nodes/manuscript.ts [hunk 3]
```diff
@@ -25,7 +25,7 @@ export interface ActualManuscriptNode extends ManuscriptNode {
 }
 
 export const manuscript: NodeSpec = {
-  content: '(section | sections)+ meta_section', // sections is a group of elements that have group: sections
+  content: 'title* (section | sections)+ meta_section', // sections is a group of elements that have group: sections
```
[the remark this answers is not in the document]
Comment (mbartenev-atypon) [reply src/schema/nodes/manuscript.ts:28]: to be adressed in LEAN-3126

File: src/jats/__tests__/jats-roundtrip.test.ts (hunk 1 above)
Comment (mbartenev-atypon) [reply src/jats/__tests__/jats-roundtrip.test.ts:167]: to be adressed in LEAN-3126

**Review by mbartenev-atypon — APPROVED** [review]

File: src/transformer/__tests__/encode.test.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (nghazzawi-atypon) [reply src/transformer/__tests__/encode.test.ts:94]: @mbartenev-atypon  
The issue has been addressed and resolved under LEAN-3030

File: src/jats/__tests__/jats-roundtrip.test.ts (hunk 1 above)
Comment (nghazzawi-atypon) [reply src/jats/__tests__/jats-roundtrip.test.ts:167]: The issue has been addressed and resolved under LEAN-3030.

File: src/schema/nodes/manuscript.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (nghazzawi-atypon) [reply src/schema/nodes/manuscript.ts:28]: updated under LEAN-3030.

File: src/transformer/__tests__/encode.test.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (nghazzawi-atypon) [reply src/transformer/__tests__/encode.test.ts:94]: The issue has been addressed and resolved under LEAN-3030.

**Pull request merged**